examples/trafficLightStopSignTest/trafficLightsStopSignTest.dbc.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version='1.0' encoding='utf-8'?>
+<criteria xmlns="http://drivebuild.com" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://drivebuild.com drivebuild.xsd">
+    <environment>test_case0.dbe.xml</environment>
+    <author>Michael Heine</author>
+    <version>1</version>
+    <participants>
+        <participant id="ego" model="ETK800">
+            <initialState x="1.0" y="0.0" orientation="0" movementMode="AUTONOMOUS" speed="50" />
+            <ai>
+                <roadCenterDistance id="egoLaneDist" />
+                <camera width="600" height="400" fov="120" direction="FRONT" id="egoFrontCamera" />
+            </ai>
+            <movement>
+                <waypoint x="1.0" y="0.0" tolerance="2" movementMode="AUTONOMOUS" />
```

Comment:
Is this duplication of the initial state position required? If so, I should fix that. Otherwise please remove this line to make clear to everyone looking at this example that it is not required.

Reply:
Do you mean the first waypoint and the initialState tag?

Reply:
Yes. The first waypoint contains the same information as the initial state.